docs/modules/ROOT/pages/how-to-guides/Secure-your-supply-chain/proc_java_dependencies.adoc

```diff
@@ -290,7 +370,7 @@ legacyRepos: <6>
 After adding this information, re-running the build should resolve this information, moving it to the  state `ArtifactBuildBuilding`, and eventually to `ArtifactBuildComplete`.
```

Comment:
i know you didn't touch a lot of this content (still in white), but since you're in it now, go ahead and fix other style/clarity issues? for example, in line 308: "has ended up in this state" is jacked.

your call!